sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -115,10 +115,11 @@ struct Adsd3500Sensor::ImplData {
 // Should not have duplicated code if possible.
 static int xioctl(int fh, unsigned int request, void *arg) {
     int r;
+    int tries = 3;
```

Comment:
Why the need to change?

Reply:
ctrl_app uses the same retry mechanism, it seems more stable.